randovania/games/cave_story/layout/cs_configuration.py

```diff
@@ -50,11 +52,16 @@ def script(self) -> str:
             return "<FL+6004<IT+0005"
         raise ValueError(f"No script for objective {self}")
 
+@dataclasses.dataclass(frozen=True)
+class HintConfiguration(BitPackDataclass, DataclassPostInitTypeCheck, JsonDataclass):
+    item_hints: bool = True
+
 @dataclasses.dataclass(frozen=True)
 class CSConfiguration(BaseConfiguration):
     puppies_anywhere: bool
     objective: CSObjective
     no_blocks: bool
+    hints: HintConfiguration = HintConfiguration()
```

Comment:
No need for default values, or more precisely, better to not have them.

Reply:
this one has a default value because it isn't intended to be configurable; stuff breaks elsewhere when assigning hints if I don't have this property in the configuration